controllers/configurationpolicy_controller.go

```diff
@@ -517,13 +517,21 @@ func (r *ConfigurationPolicyReconciler) cleanUpChildObjects(plc policyv1.Configu
 		// determine whether object should be deleted
 		needsDelete := false
 
-		existing, _ := getObject(
+		existing, err := getObject(
 			namespaced,
 			object.Object.Metadata.Namespace,
 			object.Object.Metadata.Name,
 			mapping.Resource,
 			r.TargetK8sDynamicClient,
 		)
+		if err != nil {
+			log.Error(fmt.Errorf("failed to get object"), "Error")
```

Comment:
Why not
```suggestion
			log.Error(err, "Failed to get object")
```

Reply:
In my first iteration, I wasn't using the error object because it was empty. I'll update--thanks!